src/aks-preview/HISTORY.rst

```diff
@@ -9,6 +9,10 @@ If there is no rush to release a new version, please just add a description of t
 
 To release a new version, please select a new version number (usually plus 1 to last patch version, X.Y.Z -> Major.Minor.Patch, more details in `\doc <https://semver.org/>`_), and then add a new section named as the new version number in this file, the content should include the new modifications and everything from the *Pending* section. Finally, update the `VERSION` variable in `setup.py` with this new version number.
 
+Pending
```

Comment:
Please release a new major version (e.g., 8.0.0b1) as this is a breaking change (some options got removed). Don't forget to update the value of `VERSION` in `src/aks-preview/setup.py` accordingly.